### PR TITLE
Add custom version switcher

### PR DESCRIPTION
**Pull request by robredpath**
Figured it out! 

No need to merge this any time soon, but it means everything's here for when we want it.

### CODE REVIEW

**Comment by duncandewhurst** [comment]
Thanks @robredpath 

The version switcher is working (showing live and master branches) but we'll want to merge this PR before deploying any further versions which should appear in the switcher.

**Comment by robredpath** [comment]
Yeah, there's a sensible default that works for now. But, when we want to customise it, we can merge this and then update the relevant file in opendataservices-deploy.

File: docs/_templates/layout.html [hunk 1]
```diff
@@ -10,3 +10,8 @@
     <img src="{{ pathto('_static/images/logos/') }}cost.png" style="padding-left:40px; padding-right:40px;"/>
   </a>
 {% endblock %}
+
```
Comment (jpmckinney) [inline docs/_templates/layout.html:13]: Remove extra newline, please :)

**Comment by jpmckinney** [comment]
I'm not sure why I closed this PR (perhaps accidentally?), but I made an identical commit just now.

**Pull request closed without merging**